frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemDestination/uploadListItemDestination.component.tsx

```diff
@@ -169,9 +169,9 @@ export const UploadListItemDestination = ({
 
 	return (
 		<Autocomplete
```

Comment:
I'm not entirely sure why, and it may be because the uploads form definitely needs a rework, but the destination input is now very laggy when typing. Also the box doesn't change colour on error
This issue: ![image](https://user-images.githubusercontent.com/62481250/205078905-914214ca-18e4-4e23-ac43-2897595dc499.png)
Issue-3583: ![image](https://user-images.githubusercontent.com/62481250/205079008-01e8652f-d5da-4c4a-9397-486aaf40c36e.png)

Reply:
Are you sure you pulled the latest version? I remember having this problem as well, but I fixed it in one of my latest commits and I cannot reproduce this on my branch